tests/test_power_board.py

```diff
@@ -190,14 +190,14 @@ def test_power_board_outputs(powerboard_serial: MockPowerBoard) -> None:
     power_board.outputs[PowerOutputPosition.L0].is_enabled = False
     power_board.outputs[PowerOutputPosition.L1].is_enabled = True
     power_board.outputs[PowerOutputPosition.L1].is_enabled = False
-    power_board.outputs[PowerOutputPosition.L2].is_enabled = True
-    power_board.outputs[PowerOutputPosition.L2].is_enabled = False
     power_board.outputs[PowerOutputPosition.L3].is_enabled = True
     power_board.outputs[PowerOutputPosition.L3].is_enabled = False
+    power_board.outputs[PowerOutputPosition.FIVE_VOLT].is_enabled = True
+    power_board.outputs[PowerOutputPosition.FIVE_VOLT].is_enabled = False
 
     # Test that we can't enable or disable the 5V output
```

Comment:
Nit: Comment needs updating

```suggestion
    # Test that we can't enable or disable the brain output
```